content/docs/05.concepts/04.secret.md

```diff
@@ -85,25 +70,16 @@ If you would add the environment variable to the `kestra` container section in a
   kestra:
     image: kestra/kestra:latest
     environment:
-      SECRET_MYPASSWORD: cGFzc3dvcmQ=
+      SECRET_MYPASSWORD: bXlQcml2YXRlQ29kZQ==
 ```
 
-This variable can then be used in a flow using the `{{ secret('secret_key_name') }}` syntax:
-
-```yaml
-id: secretTest
-namespace: company.team
-tasks:
-  - id: hello
-    type: io.kestra.plugin.core.log.Log
-    message: "{{ secret('MYPASSWORD') }}"
-```
+This secret can then be used in a flow using the `{{ secret('SECRET_MYPASSWORD') }}` syntax, and it will base64-decoded during flow execution. Make sure to not include the prefix `SECRET_` when calling the `secret('MYPASSWORD')` function, as this prefix is only there in the environment variable definition to prevent Kestra from treating other system variables as secrets (for better performance and increased security).
```

Comment:
```suggestion
This secret can then be used in a flow using the `{{ secret('MYPASSWORD') }}` syntax, and it will base64-decoded during flow execution. Make sure to not include the prefix `SECRET_` when calling the `secret('MYPASSWORD')` function, as this prefix is only there in the environment variable definition to prevent Kestra from treating other system variables as secrets (for better performance and increased security).
```